crates/argmin-math/src/lib.rs

```diff
@@ -131,14 +132,69 @@
 #![deny(clippy::float_cmp)]
 
 cfg_if::cfg_if! {
-    if #[cfg(feature = "nalgebra_0_32")] {
+    if #[cfg(feature = "nalgebra_0_33")] {
+        extern crate nalgebra_0_33 as nalgebra;
+        trait Allocator<T, R, C = nalgebra::U1>: nalgebra::allocator::Allocator<R, C>
+        where
+            R: nalgebra::Dim,
+            C: nalgebra::Dim,
+        {}
+        impl<T, R, C, U> Allocator<T, R, C> for U
+        where
+            U: nalgebra::allocator::Allocator<R, C>,
+            R: nalgebra::Dim,
+            C: nalgebra::Dim,
+        {}
+        trait SameShapeAllocator<T, R1, C1, R2, C2>: nalgebra::allocator::SameShapeAllocator<R1, C1, R2, C2>
+        where
+            R1: nalgebra::Dim,
+            C1: nalgebra::Dim,
+            R2: nalgebra::Dim,
+            C2: nalgebra::Dim,
+            nalgebra::constraint::ShapeConstraint: nalgebra::constraint::SameNumberOfRows<R1,R2> + nalgebra::constraint::SameNumberOfColumns<C1,C2>,
+        {}
+        impl<T, R1, C1, R2, C2, U> SameShapeAllocator<T, R1, C1, R2, C2> for U
+        where
+            U: nalgebra::allocator::SameShapeAllocator<R1, C1, R2, C2>,
+            R1: nalgebra::Dim,
+            C1: nalgebra::Dim,
+            R2: nalgebra::Dim,
+            C2: nalgebra::Dim,
+            nalgebra::constraint::ShapeConstraint: nalgebra::constraint::SameNumberOfRows<R1,R2> + nalgebra::constraint::SameNumberOfColumns<C1,C2>,
+        {}
```

Comment:
I will have to trust you on this ;)

Reply:
It's mostly copy pasted trait bounds from nalgebra :P